examples/snippets/utils/operation/job_operations.py

```diff
@@ -1,14 +1,19 @@
 """Snippet for interacting with Tamr operations (or jobs)"""
 import tamr_toolbox as tbox
 
-
 # Make Tamr Client
 tamr = tbox.utils.client.create(username="user", password="pw", host="localhost")
 
-# Create a unresolved operation, i.e., PENDING or RUNNING
+# Create an unresolved operation, i.e., PENDING or RUNNING
 project = tamr.projects.by_name("my_project_id")
 op = project.unified_dataset().refresh(asynchronous=True)
 
+# Monitor the operation
```

Comment:
Add an extra comment or two on what monitor is doing exactly. Namely, returning when there is a state change

Also for wait